src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_params.py

```diff
@@ -182,3 +186,7 @@ def _get_feature_in_preview_message():
                       default=_get_default_install_location('kubectl'))
 register_cli_argument('aks install-cli', 'client_version', options_list=('--client-version',),
                       validator=validate_k8s_client_version)
+register_cli_argument('aks install-connector', 'os_type', help='The OS type of the connector', **enum_choice_list(aci_conector_os_type))
+register_cli_argument('aks install-connector', 'chart_url', help='URL to the chart', default=aci_connector_chart_url)
+register_cli_argument('aks remove-connector', 'os_type', help='The OS type of the connector', **enum_choice_list(aci_conector_os_type))
+register_cli_argument('aks remove-connector', 'graceful', help='Mention if you want to drain/uncordon your aci-connector to move your applications', default=False)
```

Comment:
For this to actually be a flag, use `action='store_true'` then remove the default.
Right now, you'd have to specify `--graceful True` which isn't what is wanted.